ui-participant/src/hub/survey/SurveyView.test.tsx

```diff
@@ -230,16 +244,25 @@ describe('Renders a survey', () => {
 const setupSurveyTest = (survey: Survey) => {
   const submitSpy = jest.spyOn(Api, 'updateSurveyResponse')
     .mockImplementation(() => Promise.resolve(mockHubResponse()))
+  const autosaveManager = {
+    trigger: (): void => { throw 'no autosave registered' }
+  };
+
+  (useAutosaveEffect as jest.Mock).mockImplementation(saveFn => {
+    autosaveManager.trigger = () => { saveFn() }
+  })
+  const triggerAutosave = () => autosaveManager.trigger()
+
   const configuredSurvey = {
     ...mockConfiguredSurvey(),
     survey
   }
   const { RoutedComponent } = setupRouterTest(
     <PagedSurveyView enrollee={mockEnrollee()} form={configuredSurvey}
-      studyShortcode={'study'} taskId={'guid34'} autoSaveInterval={200}/>)
+      studyShortcode={'study'} taskId={'guid34'}/>)
   render(RoutedComponent)
   expect(screen.getByText('You are on page1')).toBeInTheDocument()
-  return { submitSpy, RoutedComponent }
+  return { submitSpy, RoutedComponent, triggerAutosave }
```

Comment:
Sorry for my JS ignorance, but this is returning three things, but callers are expecting two things? (It looks like this used to return 2 things and the callers were expecting 1 thing.)

Reply:
this is returning an object that used to have two properties, but now has three.  Callers are free to use any/all/none of those properties, and they are referenced by name so adding new properties doesn't impact existing callers.